app/controllers/api/v2/organisations_controller.rb

```diff
@@ -2,16 +2,24 @@ module API
   module V2
     class OrganisationsController < API::V2::ApplicationController
       def index
+        per_page = params[:page][:per_page] || 10
```

Comment:
move to a private method?

Reply:
Good shout. I've done that, and fixed the failure it was also causing. 👍